src/main/java/de/hhu/propra/sharingplatform/model/User.java

```diff
@@ -104,4 +96,14 @@ public Collection getItemsExcludeRemoved() {
         }
         return notRemovedItems;
     }
+
+    public List<Contract> getChosenContracts(boolean finished) {
```

Comment:
Why don't we use the repo to get finished contracts? The controller or service could add fetch all not finished contracts from the repo and add them to the model.
What you did is a better SRP solution. But is is much slower than the repo isn't it?